packages/block-library/src/search/edit.js

```diff
@@ -408,15 +408,28 @@ export default function SearchEdit( {
 			</BlockControls>
 
 			<InspectorControls>
-				<PanelBody title={ __( 'Settings' ) }>
-					<VStack
-						className="wp-block-search__inspector-controls"
-						spacing={ 4 }
+				<ToolsPanel
+					label={ __( 'Settings' ) }
+					resetAll={ () => {
+						setAttributes( {
+							width: undefined,
+							widthUnit: undefined,
+						} );
+					} }
+				>
+					<ToolsPanelItem
+						hasValue={ () => !! width }
+						label={ __( 'Width' ) }
+						resetAll={ () =>
+							setAttributes( {
+								width: undefined,
+							} )
+						}
 					>
 						<UnitControl
 							__next40pxDefaultSize
 							label={ __( 'Width' ) }
-							id={ unitControlInputId } // unused, kept for backwards compatibility
```

Comment:
We should leave this comment here present. It's useful to know that the unused property was kept there for a reason. Otherwise we'll forget about it and remove it.

Reply:
Above mentioned changes are added in the recent commit of this PR!